src/util/common-util.ts

```diff
@@ -6,16 +6,12 @@ import { Address, BaseAddressModel } from '@/models/Address'
 import { Role, SlipStatus } from '@/util/constants'
 
 import KeyCloakService from 'sbc-common-components/src/services/keycloak.services'
-import * as moment from 'moment-timezone'
+import moment from 'moment'
 
 export default class CommonUtils {
   // Formatting date in the desired format for displaying in the template
   static formatDisplayDate (date: Date, format?: string) {
-    if (date) {
-      return moment.tz(date, 'Canada/Vancouver').format(format || 'MMM DD, YYYY')
-    } else {
-      return ''
-    }
+    return date ? moment(date).format(format || 'MMM DD, YYYY') : ''
```

Comment:
Shouldn't you set the timezone here? Wont all the dates be in UTC?

Reply:
Probably, but I just focused on what was in the ticket (The Cheque date). Changing this impacts quite a few of the dates.